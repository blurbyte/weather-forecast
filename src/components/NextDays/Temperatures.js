import styled from 'styled-components';

import media from '../../styles/media';

const Temperatures = styled.div`
  margin-left: 2rem;
  margin-top: 0.4rem;

  ${media.tablet`
    margin: 0;
    display: flex;
    align-items: center;
    margin-left: auto;
  `};
`;

export default Temperatures;
