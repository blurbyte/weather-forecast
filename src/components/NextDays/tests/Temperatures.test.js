import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Temperatures from '../Temperatures';

test('renders correctly', () => {
  const component = renderer.create(<Temperatures />);
  expect(component.toJSON()).toMatchSnapshot();
});
