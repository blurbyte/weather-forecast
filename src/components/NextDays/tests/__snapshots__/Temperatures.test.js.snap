// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  margin-left: 2rem;
  margin-top: 0.4rem;
}

@media (max-width:40em) {
  .c0 {
    margin: 0;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    margin-left: auto;
  }
}

<div
  className="c0"
/>
`;
